### PR TITLE
reverting url migration for citizen frontnd urls

### PR DESCRIPTION
**Pull request by remya-james**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPET-419

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -67,23 +67,23 @@ withPipeline(type, product, component) {
...
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-staging.service.core-compute-aat.internal/"
   }
 
   afterAlways('smoketest:aat') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
   }
 
   before('functionalTest:preview') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.preview.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.service.core-compute-preview.internal/"
```
Comment (AlokDatta) [inline Jenkinsfile_CNP:86]: this url should be - https://prl-citizen-frontend-pr-886.preview.platform.hmcts.net

So line change 86 is correct

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -67,23 +67,23 @@ withPipeline(type, product, component) {
   }
 
   before('smoketest:preview') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.preview.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.service.core-compute-preview.internal/"
```
Comment (AlokDatta) [inline Jenkinsfile_CNP:70]: this url should be - https://prl-citizen-frontend-pr-886.preview.platform.hmcts.net/

So line change 70 is correct

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -92,7 +92,7 @@ withPipeline(type, product, component) {
   }
 
   before('functionalTest:aat') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend.aat.platform.hmcts.net/"
```
Comment (AlokDatta) [inline Jenkinsfile_CNP:95]: this must be the url - "https://prl-citizen-frontend-staging.service.core-compute-aat.internal/"

File: src/test/end-to-end/config.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ module.exports = {
     caseTypeFullDesc: 'C100 & FL401 Applications'
   },
 
-  baseUrl: process.env.PRL_CITIZEN_URL || 'https://prl-citizen-frontend.aat.platform.hmcts.net/',
```
Comment (AlokDatta) [inline src/test/end-to-end/config.js:17]: this should point to aat.

**Review by priteshkr — APPROVED** [review]

**Review by channa-patil — APPROVED** [review]

**Pull request merged**